### DISCUSSION TITLE
disable hCaptcha depending on the URL Param

### DISCUSSION THREAD

**Discussion by wafamustafa** [solspace/craft-freeform#669] [Questions & Answers] [answered]
I'm trying to disable hCaptcha depending on the URL Param and I cant seem to achieve the end result. I've updated the template code as per documentation to use "disableRecaptcha: true" but it doesn't seem to trigger the change
**Steps to reproduce**

``` twig
{% set formHandle = block.handle %}

{% if craft.request.getParam('giTest') is not null %}
    {% set disableRecaptcha = 'disableRecaptcha: true' %}
{% else %}
    {% set disableRecaptcha = 'disableRecaptcha: false' %}
{% endif %}

{{ craft.freeform.form(formHandle, {
    "id": formHandle,
    "class": "form-horizontal inline "~formHandle,
    disableRecaptcha
}).render() }}
```

**Expected behavior**
When working correctly should not show captcha

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Craft Pro 3.7.56
- Freeform Version: 3.13.20
- Freeform Edition:
- Fresh Install or Upgrade:

**Comment by kjmartens** [comment, marked as answer]
Sorry for the delay @wafamustafa.

I just tested this on my end, and using the `disableRecaptcha: true` parameter correctly disables hCaptcha Invisible. Please note that this parameter does not apply to reCAPTCHA v2 Checkbox or hCaptcha Checkbox. 🙂

**Comment by wafamustafa** [comment]
Hi @kjmartens, 

Thank you for the reply and clarification :). I was wondering if this is something that can be implemented for reCAPTCHA v2 Checkbox or hCaptcha Checkbox as well. Or is there a work around?

**Reply by kjmartens** [reply]
It's currently not possible, but you can also just remove the captcha field from the form layout for specific forms. This was the original intent, as with invisible captchas, they're applied globally and needed a way to allow some forms NOT to use it. I get what you're hoping for though, and we'll likely add support for it in the future. 🙂
